spark-board-ui/src/transformation.jsx

```diff
@@ -117,6 +119,68 @@ export function getTransformationStyle(transformationType) {
             console.log("WARN: Unknown transformation type: '" + transformationType + "'. Showing a white node")
             return ["white", TableIcon]
     }
+}
+
+/**
+ * Returns the HTML element that will render the node summary, which is the
+ * data shown below the icon.
+ *
+ * @param {string} type Transformation type. 
+ * @param {object} metadata Node metadata.
+ */
+function Summary({transformation_type, metadata}) {
+    switch(transformation_type) {
+        case "Project": return <></>;
+
+        case "Filter": return <></>;
```

Comment:
The condition, maybe?

Reply:
Maybe we can merge this change and create another issue to add the data of each transformation

Reply:
I thought about that, but it will probably be very large to show inside the node

Reply:
What about showing it truncated (if very large) and add a tooltip?

Reply:
I thought about a tooltip, but this information will be on the sidebar, so I'm not sure how useful it would be :thinking: 

But maybe we can also add tooltip data to the nodes

Reply:
However, we can create an issue instead of discussing here